app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,12 @@
+class GuestsController < ApplicationController
+	def show
+		@guest = Guest.find(params[:id])
+	end
+
+	def update
+		@guest = Guest.find(params[:id])
+    @room = Room.find(params[:room_id])
+    @guest.rooms << @room
```

Comment:
The functionality of what you're doing here is correct, we're adding an existing Room to a Guest. However, this action of _creating_ something is happening within an `update` method. I think a more RESTful approach would be to have this in its own `create` method, inside of a `RoomGuestsController`. That way, that specific controller is responsible for the CRUD actions for relationships between Rooms and Guests.

Reply:
Basically lifting this method and calling it `create` within a `RoomGuestsController` would be perfect.